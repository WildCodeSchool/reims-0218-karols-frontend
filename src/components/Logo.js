import React from "react"

const Logo = () => <img src="http://via.placeholder.com/100x100" alt="logo" />

export default Logo
