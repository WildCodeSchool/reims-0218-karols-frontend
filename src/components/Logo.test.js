import React from "react"
import ReactDOM from "react-dom"
import Logo from "./Logo"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<Logo />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<Logo />).toJSON()
  expect(tree).toMatchSnapshot()
})
