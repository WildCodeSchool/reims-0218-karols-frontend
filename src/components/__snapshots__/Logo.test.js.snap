// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  alt="logo"
  src="http://via.placeholder.com/100x100"
/>
`;
